### PR TITLE
Schlorrgo Kicking

### PR DESCRIPTION
**Pull request by Geevies**
* You now harmlessly kick Schlorrgo on harm intent.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Doxxmedearly** [comment]
Any special messages about harmlessly bouncing off structures, walls, etc?

File: code/modules/mob/living/simple_animal/friendly/adhomai.dm [hunk 1]
```diff
@@ -128,4 +128,23 @@
...
+
+	user.do_attack_animation(src)
+	if(left_leg?.is_usable() && right_leg?.is_usable())
+		user.visible_message(SPAN_WARNING("[user] punts \the [src]!"))
+		make_noise()
+		throw_at(get_edge_target_turf(user, get_dir(user, src)), 4, 1)
+	else
+		apply_damage(harm_intent_damage, BRUTE, used_weapon = "Attack by [user.name]") // only non-kicking attacks do damage
+		user.visible_message(SPAN_WARNING("[user] [response_harm] \the [src]!"))
+	poke(TRUE)
```
Comment (Doxxmedearly) [inline code/modules/mob/living/simple_animal/friendly/adhomai.dm:145]:
```suggestion
	if(left_leg?.is_usable() && right_leg?.is_usable())
		user.visible_message(SPAN_WARNING("[user] punts \the [src]!"))
		user.do_attack_animation(src)
		make_noise()
		throw_at(get_edge_target_turf(user, get_dir(user, src)), 4, 1)
		poke(TRUE)
	else
		..()
```

File: code/modules/mob/living/simple_animal/friendly/adhomai.dm [hunk 2]
```diff
@@ -128,4 +128,22 @@
...
+		user.visible_message(SPAN_WARNING("[user] punts \the [src]!"))
+		user.do_attack_animation(src)
+		make_noise()
+		throw_at(get_edge_target_turf(user, get_dir(user, src)), 4, 1)
+		poke(TRUE)
+	else
+		..()
+
+/mob/living/simple_animal/schlorrgo/turf_collision(var/turf/T, var/speed = THROWFORCE_SPEED_DIVISOR)
+	visible_message(SPAN_WARNING("[src] bounces off \the [T]!"))
```
Comment (Geevies) [inline code/modules/mob/living/simple_animal/friendly/adhomai.dm:147]:
```suggestion
	visible_message(SPAN_WARNING("[src] harmlessly bounces off \the [T]!"))
```

File: code/modules/mob/living/simple_animal/friendly/adhomai.dm [hunk 3]
```diff
@@ -128,4 +128,22 @@
...
+		user.do_attack_animation(src)
+		make_noise()
+		throw_at(get_edge_target_turf(user, get_dir(user, src)), 4, 1)
+		poke(TRUE)
+	else
+		..()
+
+/mob/living/simple_animal/schlorrgo/turf_collision(var/turf/T, var/speed = THROWFORCE_SPEED_DIVISOR)
+	visible_message(SPAN_WARNING("[src] harmlessly bounces off \the [T]!"))
+	playsound(T, 'sound/effects/bangtaper.ogg', 50, 1, 1) //so it plays sounds on the turf instead, makes for awesome carps to hull collision and such
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/simple_animal/friendly/adhomai.dm:148]:
```suggestion
	playsound(T, 'sound/effects/bangtaper.ogg', 50, 1, 1)
```

**Pull request merged**